config/routes.rb

```diff
@@ -3,5 +3,6 @@
 
   namespace :v1 do
     resources :projects, only: crud
+    resources :tasks, only: %i(index show create)
```

Comment:
You chose to use a standard route rather than a nested one here. Neither is right or wrong, but I'm curious as to why you made that decision.  Note, I would ask you the same thing if you made it nested. I'm just looking for thoughts in your choices.